recipe/meta.yaml

```diff
@@ -19,18 +19,24 @@ build:
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
+    - python >=3.6
     - pyyaml
     - click
+    - pandoc >=2.11.0.4
 
 test:
   imports:
     - panflute
   commands:
+    - pip check
```

Comment:
Why `pip check`?

Reply:
Probably generated automatically. It doesn't hurt though.

Reply:
On 2nd thought, I think they add it here automatically because they want to make sure there's no packaging error here. i.e. the packages installed here matches what the original author expected.